images/win/scripts/Installers/Install-PyPy.ps1

```diff
@@ -83,35 +67,29 @@ function Install-PyPy
 }
 
 # Get PyPy content from toolset
-$pypyTools = Get-ToolsetContent | Select-Object -ExpandProperty toolcache | Where-Object Name -eq "PyPy"
+$toolsetVersions = (Get-ToolsetContent).toolcache | Where-Object Name -eq "PyPy"
 
-# Get PyPy versions from the repo
-$pypyVersions = Get-PyPyVersions
+# Get PyPy releases
+$pypyVersions = Invoke-RestMethod https://downloads.python.org/pypy/versions.json
 
 Write-Host "Starting installation PyPy..."
-foreach($pypyTool in $pypyTools)
+foreach($toolsetVersion in $toolsetVersions.versions)
 {
-    foreach($pypyVersion in $pypyTool.versions)
+    # Query latest PyPy version
+    $latestMajorPyPyVersion = ($pypyVersions |
+        Where-Object {$_.python_version.StartsWith("$toolsetVersion") -and $_.stable -eq $true} |
+        Select-Object -First 1).files |

```

Comment:
Let's move .files to  `-ExpandProperty files` param and remove ()

Reply:
Done